lib/src/components/accordion/accordion_data.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+
+class DSFRAccordionData {
```

Comment:
When declaring a class with a `const` constructor do not forget to also annotate it with `@immutable`.